app/src/main/java/com/android/shelfLife/viewmodel/overview/HouseholdCreationScreenViewModel.kt

```diff
@@ -138,12 +173,12 @@ class HouseholdCreationScreenViewModel(
           }
         }
       }
-    }
-    if (shouldUpdateRepo) {
-      viewModelScope.launch {
+      if (shouldUpdateRepo) {
```

Comment:
This area needs refactoring however an update is in the works and will be integrated once the code is on main